tmpl/sql-results.handlebars

```diff
@@ -92,9 +92,9 @@ table.sql-schema-table .row-count {
         <table class="sql-schema-table" data-table-name="{{name}}">
         <thead>
         {{#if hasSingleRow}}
-            <th><a href="javascript:void(0)">{{name}}</a> <span class="row-count">{{rowCount}} row</span></th>
+            <th><a href="javascript:void(0)">{{name}}</a> <span class="row-count">{{rowCount}} {{rowMsg}}</span></th>
```

Comment:
@pamelafox sorry for commenting late here, but I am not sure this code will handle i18n multiple plurals properly. On the other hand, I am not sure you *can* handle multiple plurals in this framework. Let me know if you need more info.

Reply:
Sure, once the strings are on Crowdin I'll be able to make a better
judgement, but it seems to me that this will generate two separate strings,
which is not correct. Caroline has been dealing with the multiple plural
issue a lot so she can probably set you up with sbd. in the know.